draft-irtf-cfrg-hpke.md

```diff
@@ -798,6 +817,21 @@ security models and assumptions, and do not consider attackers capable of quantu
 computation. A full proof of post-quantum security would need to take this
 difference into account, in addition to simply using a post-quantum KEM.
 
+## Domain Separation
+
+\[\[RFC editor: please change "RFCXXXX" to the correct number before
+publication.]]
+
+The KeySchedule procedure includes the domain separation string "RFCXXXX" in
+each Expand invocation. This ensures any secrets derived in HPKE are independent
+from those used in other protocols, even when derived from the same IKM (secret).
+This domain separation does not protect the KeySchedule 'secret' from use in other
+protocols.
+
```

Comment:
`This domain separation does not protect the KeySchedule 'secret' from use in other
protocols.`: maybe this sentence is not needed, because `secret` is not exposed by the protocol.

Reply:
Or maybe we add that clarification? 

Derivation of the KeySchedule 'secret' does not include domain separation as it is an intermediate value not exposed by the protocol.

Reply:
Yes, sounds good.